### ISSUE TITLE
weight titles

### ISSUE THREAD

**Issue by dgreisen-cfpb** [cfpb/github-wiki-search#36]
It seems to me titles are more important than content. Imagine we have a document with a title "foo" the homepage has a link to this page, the link is called "foo". If someone searches for "foo" both of these documents currently have the same weight. But if we weight an h1 at 7x all the way down to an h6 at 2x, then we better capture, I think, the appropriate weights.

**Comment by OrlandoSoto** [comment]
yes, this would make the results page more usable by displaying the heavier weights at the top
:+1:
